### PR TITLE
Post: some news about QChat

### PR DESCRIPTION
**Pull request by gounux**

### CODE REVIEW

File: content/posts/2024-11-28_QChat_news.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+authors:
+  - gallaman
+categories:
+  - Announcements
```
Comment (Guts) [inline content/posts/2024-11-28_QChat_news.md:5]:
```suggestion
  - Article
```

File: content/posts/2024-11-28_QChat_news.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+comments: true
+date: 2024-11-28
+description: "Some news about QChat and the features provided by the new 1.1 version. #GISChat #GISTribe"
+icon: material/chat
+license: beerware
+links:
+  - QTribu documentation: https://qtribu.geotribu.fr/
+  - Initial announcement: https://blog.geotribu.net/2024/10/18/introducing-qchat-a-chat-within-qgis/
```
Comment (Guts) [inline content/posts/2024-11-28_QChat_news.md:13]: I think related hyperlink is better. See https://squidfunk.github.io/mkdocs-material/tutorials/blogs/basic/#related-links

File: content/posts/2024-11-28_QChat_news.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+pin: true
```
Comment (Guts) [inline content/posts/2024-11-28_QChat_news.md:14]:
```suggestion
pin: false
```

File: content/posts/2024-11-28_QChat_news.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    - QTribu
+title: "Some news about QChat, the chat within QGIS"
+subtitle:
+---
+
+# Some news about QChat, the chat within QGIS
+
+![icône globe speech GIS Chat - Credits: Global Market by DARAYANI from Noun Project (CC BY 3.0)](https://cdn.geotribu.fr/img/logos-icones/divers/globe_speech_GISChat.svg){: .img-thumbnail-left }
+
+Last month [we were announcing with a blog post "QChat"](./2024-10-18_introducing-QChat-chat-rooms-in-QGIS.md), the plugin for chatting with the other users, directly inside QGIS.
```
Comment (Guts) [inline content/posts/2024-11-28_QChat_news.md:29]:
```suggestion
Last month [we introduced "QChat"](./2024-10-18_introducing-QChat-chat-rooms-in-QGIS.md), the plugin for chatting with other QGIS users, directly inside QGIS.
```

**Review by Guts — CHANGES_REQUESTED** [review]

File: content/posts/2024-11-28_QChat_news.md (hunk 2 above)
Comment (gounux) [reply content/posts/2024-11-28_QChat_news.md]: The CI build job seems to fail, after trying "absolute" and relative link

Comment (Guts) [reply content/posts/2024-11-28_QChat_news.md]: weird. And locally?

Comment (gounux) [reply content/posts/2024-11-28_QChat_news.md]: Locally, I don't get any error when building the website with the `--strict` option.
I'm using the `mkdocs-free.yml` config :thinking:, and the `Related links` are not present under the `Metadata` section in the left part of the page

Comment (gounux) [reply content/posts/2024-11-28_QChat_news.md]: Ok @Guts, fixed :tada: 

Actually, the trick is that the external links "root" is the `content` folder, [this commit](https://github.com/geotribu/english-blog/pull/13/commits/feacaa5ddfcea9ddf96902a93a52e35705a6234a) solves it

**Review by Guts — APPROVED** [review]

**Pull request merged**